### ISSUE TITLE
better linking from api docs to examples

### ISSUE THREAD

**Issue by tacaswell** [NSLS-II/bluesky#618]
I think we may have to write a sphinx extension to do this :disappointed: .

**Comment by stan-dot** [comment]
is this out of scope? 

at the moment `docs/multi_run_plans.py` has this:
```
Plans with Sequential Runs
---------------------------

Sequential calling of multiple runs is supported by older versions of Bluesky. There is no need
to use multi-run plan features if runs are not overlapping (the next run is opened only after
the previous run is closed), but run keys still can be assigned to all or some runs if needed.

In the following example, two preassembled plans are called in sequence. Run Engine is subscribed to
a single instance of BestEffortCallback, which is set up to display data specific for each run
when the run opened.

.. literalinclude:: /examples/multi_run_plans_sequential.py

.. ipython:: python
    :suppress:

    %run -m multi_run_plans_sequential

.. ipython:: python

    RE(plan_sequential_runs(10))
    
    ```
    
    with `.. literallinculde::` lines added 4 years ago (2020). 
    
 is this already implemented then @tacaswell ?
